### ISSUE TITLE
Some characters on ATB

### ISSUE THREAD

**Issue by ghost** [tschw/AntTweakBar#5]
Hi there, I am working on a project using anttweakbar in it, my project language is Turkish.
As far I see, AntTweakBar does not supports most of Turkish letters.
Is there a way to fix atb for Turkish letters?

Thanks.

**Comment by lucifer602288** [comment]
Yes, The fonts are in pgm files in directory \TweakBar\compile\res saved. Create a pgm file for your language and add it to the code e.g. spanish, norwegian, cyrillic, indonesian, arrabic, devanagari, etc.

**Comment by tschw** [comment]
An alternative approach for the less artistic engineer could be to replace bitmap font handling with something that renders standard vector fonts, this way supporting any character therein (or at least the ones on the Unicode's BMP).

**Comment by ghost** [comment]
Hmm, thanks for answers.
